### PR TITLE
Fix cache

### PR DESCRIPTION
**Pull request by metanerd**

### CODE REVIEW

**Review by jaydeland — APPROVED** [review]
Lgtm

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -7,9 +7,13 @@ include:
   file: private.yml
 
 cache:
-  key: ${CI_COMMIT_REF_SLUG}
+  key:
+    files:
+      - package.json
```
Comment (hanzei) [inline .gitlab-ci.yml:12]: Should we also check for changes in  `package-lock.json`? E.g. security updates could only update `package-lock.json`.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -7,9 +7,13 @@ include:
...
+    prefix: ${CI_COMMIT_REF_SLUG}
   paths:
     - .npm/
```
Comment (hanzei) [inline .gitlab-ci.yml:15]: I'm yet to encounter a `.npm` directory. What is it used for?

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for taking a look at the issue!

Should we merge this into `master` directly to ensure other don't run into the same issue?

File: .gitlab-ci.yml (hunk 2 above)
Comment (metanerd) [reply .gitlab-ci.yml:15]: https://docs.gitlab.com/ee/ci/caching/#cache-nodejs-dependencies

File: .gitlab-ci.yml (hunk 1 above)
Comment (metanerd) [reply .gitlab-ci.yml:12]: `package.json` is fine for now, we can add package-lock.json when the issue arises.

**Review by jaydeland — COMMENTED** [review]
Lgtm

**Comment by metanerd** [comment]
We can merge it into master with your PR @hanzei

**Review by hanzei — APPROVED** [review]

**Pull request merged**